sdk/keyvault/azure-security-keyvault-administration/pom.xml

```diff
@@ -50,6 +50,11 @@
       <artifactId>azure-core</artifactId>
       <version>1.38.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-json</artifactId>
+      <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-json;dependency} -->
```

Comment:
Do we need to explicitly add this? Isn't azure-core transitively bringing this?

Reply:
It does but this reduces the chances that dependency resolution changes the version due to transitive dependencies.